ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.19-upgrade-from-stable-4.18.yaml

```diff
@@ -42,12 +42,14 @@ base_images:
 releases:
   initial:
     prerelease:
+      architecture: multi
```

Comment:
Reviewing the doc, we already have entries for this configuration called `multi-latest` and `multi-initial`.  Maybe we can test a little by updating the job definition to target those instead of just `initial` and `latest` just for testing?

Reply:
@deepsm007 keep me honest here.

`multi-initial` and `multi-latest` are arbitrary variables that don't hold any significance to ci-operator.

`initial` and `latest` settings do. ci-operator will extract the installer and openshift-tests (maybe oc as well?) from the payload defined here.

Reply:
correct